src/clients/InsightsQueryClient.ts

```diff
@@ -102,6 +102,18 @@ class InsightsQueryClient {
     return response.data as Insights.InsightsQueryResponse<Insights.QueryGetPoolsVolumeResponse>
   }
 
+  async PoolsLiquidity(req: Insights.QueryGetPoolsLiquidityRequest = {}): Promise<Insights.InsightsQueryResponse<Insights.QueryGetPoolsLiquidityResponse>> {
+    const queryParams = {
+      market: req.market,
```

Comment:
- PoolsLiquidity url does not take `market` query param. It takes an array of pool ids, i.e. `?poolId=1,2,3`. Make sure to check the corresponding query on `carbon-stats` project to get the correct query params
https://github.com/Switcheo/carbon-stats/blob/master/src/routes/pool/liquidity.ts

- Be mindful what params you are passing into the `apiManager`. Here, you are directly passing an array of markets (e.g. `['swth_eth', 'eth_dai']`). However, the resulting url becomes `https://api-insights.carbon.network/pool/liquidity?market=swth_eth&market=eth_dai` which causes the filter to not work.